.github/workflows/ci.yaml

```diff
@@ -0,0 +1,157 @@
+          path: ${{env.WORKING_DIRECTORY_APP}}/build
+
+      - name: Download Docker ${{env.SENTENCE_PRODUCER_IMAGE_NAME}} tar artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{env.SENTENCE_PRODUCER_IMAGE_NAME}}
+          path: ${{env.WORKING_DIRECTORY_APP}}/build
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Publish Docker image
+        run: |
+          if [[ $(ls -1 build/*.tar 2>/dev/null | wc -l) != 2 ]]; then
+            >&2 echo "Error: images tar files are needed in the downloaded artifact. You can upload them before using this action: https://github.com/actions/upload-artifact."
+            exit 1
+          fi
+          docker load --input build/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}.tar
+          docker load --input build/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}.tar
+          if [[ "$GITHUB_REF" =~ ^refs/tags/.* ]]; then
+            # SENTENCE_PRODUCER
+            docker tag ${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${{ github.run_id }} ${{ env.DOCKER_REGISTRY }}/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:latest
+            docker tag ${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${{ github.run_id }} ${{ env.DOCKER_REGISTRY }}/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${GITHUB_REF/refs\/tags\//}
+            docker push ${{ env.DOCKER_REGISTRY }}/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:latest
+            docker push ${{ env.DOCKER_REGISTRY }}/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${GITHUB_REF/refs\/tags\//}
+
+            # WORD_COUNT_APP
+            docker tag ${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${{ github.run_id }} ${{ env.DOCKER_REGISTRY }}/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:latest
+            docker tag ${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${{ github.run_id }} ${{ env.DOCKER_REGISTRY }}/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${GITHUB_REF/refs\/tags\//}
+            docker push ${{ env.DOCKER_REGISTRY }}/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:latest
+            docker push ${{ env.DOCKER_REGISTRY }}/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${GITHUB_REF/refs\/tags\//}
+          else
+            # SENTENCE_PRODUCER
+            docker tag ${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${{ github.run_id }} ${{ env.DOCKER_REGISTRY }}/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${{ github.run_id }}
+            docker push ${{ env.DOCKER_REGISTRY }}/${{env.SENTENCE_PRODUCER_IMAGE_NAME}}:${{ github.run_id }}
+
+            # WORD_COUNT_APP
+            docker tag ${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${{ github.run_id }} ${{ env.DOCKER_REGISTRY }}/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${{ github.run_id }}
+            docker push ${{ env.DOCKER_REGISTRY }}/${{env.WORD_COUNT_APPLICATION_IMAGE_NAME}}:${{ github.run_id }}
+          fi
+        shell: bash
+        working-directory: ${{env.WORKING_DIRECTORY_APP}}
```

Comment:
why not just
```
# deploy producer app
gradle jib -Djib.to.image=bakdata/kpops-demo-sentence-producer -Djib.container.mainClass=com.bakdata.kpops.examples.SentenceProducer


# deploy streams-app
gradle jib -Djib.to.image=bakdata/kpops-demo-word-count-app-Djib.container.mainClass=com.bakdata.kpops.examples.WordCountApplication
```

Reply:
Then you cannot control the tags

Reply:
Why not?
```
# deploy producer app
gradle jib -Djib.to.image=bakdata/kpops-demo-sentence-producer:tag1 -Djib.container.mainClass=com.bakdata.kpops.examples.SentenceProducer


# deploy streams-app
gradle jib -Djib.to.image=bakdata/kpops-demo-word-count-app:tag2 -Djib.container.mainClass=com.bakdata.kpops.examples.WordCountApplication

```

Reply:
We want to set multiple tags. latest and version for tag builds e.g.

Reply:
But I don't understand why we can't use the flag `-Djib.to.tags=a,b,c`
```
gradle jib -Djib.to.image=bakdata/kpops-demo-word-count-app -Djib.container.mainClass=com.bakdata.kpops.examples.WordCountApplication
-Djib.to.tags=latest,version-tag,bla
```

Reply:
That should work

Reply:
@irux Then please remove all the docker commands and directly use the command above to build, tag, and push.

Reply:
like this @raminqaf